README.md

```diff
@@ -626,7 +626,7 @@ cdnvm() {
 }
 
 alias cd='cdnvm'
-cdnvm "$PWD" || exit
```

Comment:
$PWD is in posix, afaik; which systems don't come with it?

if the command isn't here, then a new shell won't have the logic applied, so it does have to remain.

Reply:
for some reason it doesn't come by default on pop os, and kubuntu

Reply:
Perhaps those shells aren’t POSIX-compliant, then?